karaf/jetty-config/src/main/resources/jetty.xml

```diff
@@ -56,6 +56,9 @@
                         <Item>org.apache.brooklyn.rest.security.jaas.BrooklynLoginModule$RolePrincipal</Item>
                     </Array>
                 </Set>
+                <Set name="identityService">
+                    <New class="org.eclipse.jetty.security.DefaultIdentityService"/>
+                </Set>
```

Comment:
That's the default if none is specified (see [code](https://github.com/eclipse/jetty.project/blob/jetty-9.3.x/jetty-jaas/src/main/java/org/eclipse/jetty/jaas/JAASLoginService.java#L178)).